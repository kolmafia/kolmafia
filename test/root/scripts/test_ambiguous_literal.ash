$location[dire warren];
$item[staff of ed];
