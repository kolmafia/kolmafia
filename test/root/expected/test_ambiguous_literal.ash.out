Changing "dire warren" to "The Dire Warren" would get rid of this message. (test_ambiguous_literal.ash, line 1)
Multiple matches for "staff of ed"; using "[7961]Staff of Ed". (test_ambiguous_literal.ash, line 2) Clarify by using one of:
$item[[2325]Staff of Ed]
$item[[7961]Staff of Ed]
